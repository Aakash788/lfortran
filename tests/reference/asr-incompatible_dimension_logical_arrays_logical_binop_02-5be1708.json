{
    "basename": "asr-incompatible_dimension_logical_arrays_logical_binop_02-5be1708",
    "cmd": "lfortran --show-asr --no-color {infile} -o {outfile}",
    "infile": "tests/errors/incompatible_dimension_logical_arrays_logical_binop_02.f90",
    "infile_hash": "4f0b876fc6fd7aa0f609f1ed921aad63b1ec570ad4c91f700f81325f",
    "outfile": null,
    "outfile_hash": null,
    "stdout": "asr-incompatible_dimension_logical_arrays_logical_binop_02-5be1708.stdout",
    "stdout_hash": "a67dff9ee7f2c1146e64ebbc0c7000af6385caa61999596231e44c4a",
    "stderr": null,
    "stderr_hash": null,
    "returncode": 0
}