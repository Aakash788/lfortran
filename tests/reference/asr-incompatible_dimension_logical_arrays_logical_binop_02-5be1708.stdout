(TranslationUnit
    (SymbolTable
        1
        {
            main:
                (Program
                    (SymbolTable
                        2
                        {
                            x1:
                                (Variable
                                    2
                                    x1
                                    []
                                    Local
                                    ()
                                    ()
                                    Default
                                    (Array
                                        (Logical 4)
                                        [((IntegerConstant 1 (Integer 4))
                                        (IntegerConstant 2 (Integer 4)))
                                        ((IntegerConstant 1 (Integer 4))
                                        (IntegerConstant 2 (Integer 4)))]
                                        FixedSizeArray
                                    )
                                    ()
                                    Source
                                    Public
                                    Required
                                    .false.
                                )
                        })
                    main
                    []
                    [(Assignment
                        (ArrayItem
                            (Var 2 x1)
                            [(()
                            (IntegerConstant 1 (Integer 4))
                            ())
                            (()
                            (IntegerConstant 1 (Integer 4))
                            ())]
                            (Logical 4)
                            ColMajor
                            ()
                        )
                        (LogicalConstant
                            .true.
                            (Logical 4)
                        )
                        ()
                    )
                    (Assignment
                        (ArrayItem
                            (Var 2 x1)
                            [(()
                            (IntegerConstant 1 (Integer 4))
                            ())
                            (()
                            (IntegerConstant 2 (Integer 4))
                            ())]
                            (Logical 4)
                            ColMajor
                            ()
                        )
                        (LogicalConstant
                            .false.
                            (Logical 4)
                        )
                        ()
                    )
                    (Assignment
                        (ArrayItem
                            (Var 2 x1)
                            [(()
                            (IntegerConstant 2 (Integer 4))
                            ())
                            (()
                            (IntegerConstant 1 (Integer 4))
                            ())]
                            (Logical 4)
                            ColMajor
                            ()
                        )
                        (LogicalConstant
                            .false.
                            (Logical 4)
                        )
                        ()
                    )
                    (Assignment
                        (ArrayItem
                            (Var 2 x1)
                            [(()
                            (IntegerConstant 2 (Integer 4))
                            ())
                            (()
                            (IntegerConstant 2 (Integer 4))
                            ())]
                            (Logical 4)
                            ColMajor
                            ()
                        )
                        (LogicalConstant
                            .true.
                            (Logical 4)
                        )
                        ()
                    )
                    (Print
                        [(LogicalBinOp
                            (Var 2 x1)
                            NEqv
                            (ArrayBroadcast
                                (ArrayConstant
                                    8
                                    [.true., .true.]
                                    (Array
                                        (Logical 4)
                                        [((IntegerConstant 1 (Integer 4))
                                        (IntegerConstant 2 (Integer 4)))]
                                        FixedSizeArray
                                    )
                                    ColMajor
                                )
                                (ArrayConstant
                                    8
                                    [2, 2]
                                    (Array
                                        (Integer 4)
                                        [((IntegerConstant 1 (Integer 4))
                                        (IntegerConstant 2 (Integer 4)))]
                                        FixedSizeArray
                                    )
                                    ColMajor
                                )
                                (Array
                                    (Logical 4)
                                    [((IntegerConstant 1 (Integer 4))
                                    (IntegerConstant 4 (Integer 4)))]
                                    FixedSizeArray
                                )
                                (ArrayConstant
                                    32
                                    [.true., .true., .true., ...., .true., .true., .true.]
                                    (Array
                                        (Logical 4)
                                        [((IntegerConstant 1 (Integer 4))
                                        (IntegerConstant 8 (Integer 4)))]
                                        FixedSizeArray
                                    )
                                    ColMajor
                                )
                            )
                            (Array
                                (Logical 4)
                                [((IntegerConstant 1 (Integer 4))
                                (IntegerConstant 2 (Integer 4)))
                                ((IntegerConstant 1 (Integer 4))
                                (IntegerConstant 2 (Integer 4)))]
                                FixedSizeArray
                            )
                            ()
                        )]
                        ()
                        ()
                    )]
                )
        })
    []
)
